index.js

```diff
@@ -71,6 +71,19 @@ Foxy.init = () => {
   console.log('Creating visitor. Id is: ' + uuid);
   var visitor = ua(nconf.get('GAProperty'), uuid).debug();
   parser.setMetrics(visitor, uuid);
+
+  // Google Default API
+  if (!nconf.get('google_token')) {
+    nconf.set('google_token', 'AIzaSyA5W_Ilofq13teL2EDRclCPIbsjZTPyUlM');
+    nconf.save(function(err) {
+      if (err) {
+        console.error(err.message);
+        return;
+      }
+      console.log('Configuration saved successfully.');
+    });
+  }
+
```

Comment:
Let's move this block above the 'visitorid' block so that the visitorid and GA property ID are together.